### PR TITLE
[4.0] replace btn-xs with btn-sm

### LINKED ISSUE #21244: [4.0] Bootstrap 4 dropped btn-xs buttons.
### What needs to be fixed

CF : https://getbootstrap.com/docs/4.0/migration/#buttons
> Dropped the .btn-xs class entirely as .btn-sm is proportionally much smaller than v3’s.

### Why this should be fixed

The PR #15658 already migrated buttons from BS2 to BS4 ; but since then, there was some modifications in Bootstrap (it was an alpha).

### How would you fix it

Looks like the `btn-xs` should be migrated into `btn-sm`

### Side Effects expected

These btn-xs buttons are displayed as regular big buttons.
For example, there are the " grid.order " buttons.

### PR DESCRIPTION
**Pull request by brianteeman**
In Bootstrap 4 they "dropped the .btn-xs class entirely as .btn-sm is proportionally much smaller than v3’s."

A previous PR added the css back for the frontend template.

This PR removes that additional class and instead replaces the uses of btn-xs with btn-sm

Also removes the markup pull-xs-right which doesn't exist in the css. No point giving it the float-right as it looks really silly

Fixes #21244

### CODE REVIEW

File: templates/cassiopeia/scss/vendor/bootstrap/_buttons.scss [hunk 1]
```diff
@@ -1,18 +1,4 @@
 // Buttons
```
Comment (Quy) [inline templates/cassiopeia/scss/vendor/bootstrap/_buttons.scss:1]:
```suggestion
// Buttons

```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9581a54532bbdb66506bc8052822dd47ebb31bb6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24979">issues.joomla.org/tracker/joomla-cms/24979</a>.</sub>

**Comment by brianteeman** [comment]
Updating branch to trigger drone

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**